pkg/kubelet/apis/config/validation/validation_test.go

```diff
@@ -74,31 +74,17 @@ var (
 )
 
 func TestValidateKubeletConfiguration(t *testing.T) {
-	successCases := []struct {
+	cases := []struct {
 		name      string
 		configure func(config *kubeletconfig.KubeletConfiguration) *kubeletconfig.KubeletConfiguration
+		errMsg    string
 	}{
 		{
 			name: "Success",
 			configure: func(conf *kubeletconfig.KubeletConfiguration) *kubeletconfig.KubeletConfiguration {
 				return conf
 			},
 		},
-	}
-
-	for _, tc := range successCases {
-		t.Run(tc.name, func(t *testing.T) {
-			if errs := validation.ValidateKubeletConfiguration(tc.configure(successConfig.DeepCopy())); errs != nil {
-				t.Errorf("unexpected error: %v", errs)
-			}
-		})
-	}
```

Comment:
We lost the success case, we still need to test the happy path.

Reply:
It's here! 👍 

https://github.com/shuheiktgw/kubernetes/blob/ce20835107927ab5a19a72b98b42808dc2babe6b/pkg/kubelet/apis/config/validation/validation_test.go#L83-L86

Reply:
Ah I see it now, my diff might have been wonky